### PR TITLE
chore(release): Bump to version 0.2.0

### LINKED ISSUE #2123: Disallow adding the same bucket more than once
This is allowed, currently:
```
scoop bucket add comp500 https://github.com/comp500/scoop-comp500.git
scoop bucket add anothername https://github.com/comp500/scoop-comp500.git
```
We should fail with a message, like:
```
> scoop bucket add anothername https://github.com/comp500/scoop-comp500.git
ERROR: Bucket comp500 already exists for https://github.com/comp500/scoop-comp500.git
```

### LINKED ISSUE #2230: Special characters in Windows user names will cause command scripts in "shims" to fail.
My Windows username, automatically created when logging into Microsoft at install time, contains the letter 'ø'.

This is the real path of my recently installed application:
`c:\Users\Bjørn\scoop\apps\sbt\current\bin\`

This is the contents of the shim command:
`@"C:\Users\Bj?rn\scoop\apps\sbt\current\bin\sbt.bat"  %*`

When trying to run my application, it gives the following message:
```
>sbt
The system cannot find the path specified.
```

### LINKED ISSUE #2573: Autoupdate error when used directory name with multibyte characters
I create custeom app-manifest([here](https://github.com/yukihane/scoop-bucket-yukihane-games/blob/91d960886de11fc41137918394b9de2f943da616/densya.json#L7)).

Following error occurs when updating, using manifest generated by checkver.ps1.

It seems that "extract_dir" directory name is garbled, and this problem is solved by changing encoding to UTF-8 with BOM.

> <pre>
> $ scoop update densya
> densya: 1.04 -> 1.06
> Updating one outdated app:
> Updating 'densya' (1.04 -> 1.06)
> Uninstalling 'densya' (1.04)
> Removing shim for 'densya'.
> Unlinking ~\scoop\apps\densya\current
> Installing 'densya' (1.06) [64bit]
> Loading dcnt.cgi from cache
> Checking hash of dcnt.cgi ... ok.
> Extracting densya.zip ... DEBUG
> -------------------------------------------------------------------------------
>    ROBOCOPY     ::     Windows の堅牢性の高いファイル コピー
> -------------------------------------------------------------------------------
> 
>   開始: 2018年9月9日 9:24:35
>    コピー元 : C:\Users\yuki\scoop\apps\densya\1.06\_tmp\繝・Φ繧ｷ繝｣\
>      コピー先 : C:\Users\yuki\scoop\apps\densya\1.06\
> 
>     ファイル: *.*
> 
>   オプション: *.* /S /E /DCOPY:DA /COPY:DAT /MOVE /R:1000000 /W:30
> 
> ------------------------------------------------------------------------------
> 
> 2018/09/09 09:24:35 エラー 2 (0x00000002) コピー元ディレクトリにアクセスしています C:\Users\yuki\scoop\apps\densya\1.06\_tmp\繝・Φ繧ｷ繝｣\
> 指定されたファイルが見つかりません。
> 
> ERROR Could not find '繝・Φ繧ｷ繝｣'! (error 16)
> 
> Please create a new issue by using the following link and paste your console output:
> https://github.com/lukesampson/scoop/issues/new?title=densya%401.06%3a+extract_dir+error
> True
> </pre>

### LINKED ISSUE #3418: move all the libs importing to the entry bin script
I found that the mode of importing lib scripts from different libexec/lib scripts causes a really bad issue - some of the libs are imported and executed many times. This can cause other potential problems and effect performance probably.

**CASE**

1. In the `bin\scoop.ps1` entry bin script, it imported `lib\core.ps1` and `lib\bucket.ps1`::

https://github.com/lukesampson/scoop/blob/2db651c13a83f1c0ebb53723716efb5d96018fc0/bin/scoop.ps1#L7-L9

2. In the `libexec\scoop-bucket.ps1` script, it imported `lib\core.ps1` and `lib\bucket.ps1`:

https://github.com/lukesampson/scoop/blob/2db651c13a83f1c0ebb53723716efb5d96018fc0/libexec/scoop-bucket.ps1#L22-L23

3. And in the `lib\bucket.ps1` script, it also imported `lib\core.ps1`:

https://github.com/lukesampson/scoop/blob/2db651c13a83f1c0ebb53723716efb5d96018fc0/lib/buckets.ps1#L1

So, when I run `scoop bucket list`, the core lib `lib\core.ps1` is actually imported by **FOUR** times, and scoop will execute all of the code of `lib\core.ps1` by four times. You can simply add a line of code to verify it:

```diff
diff --git a/lib/core.ps1 b/lib/core.ps1
index 148f12539..6e6c54df9 100644
--- a/lib/core.ps1
+++ b/lib/core.ps1
@@ -10,6 +10,8 @@ if((test-path $oldscoopdir) -and !$env:SCOOP) {
     $scoopdir = $oldscoopdir
 }
 
+Write-Host "Hi"
+
 $globaldir = $env:SCOOP_GLOBAL, "$env:ProgramData\scoop" | Select-Object -first 1
 
 # Note: Setting the SCOOP_CACHE environment variable to use a shared directory
```

Then you will see 4 line of `Hi` printed before the bucket list.

```
$ scoop bucket list  
Hi
Hi
Hi
Hi
extras
main
```

Holy crap! And you will see all other libexec scripts are suffering this issue.

To be honest, I have realized this problem of scoop before I started to maintain pshazz. I have released a [patch](https://github.com/lukesampson/pshazz/commit/c1e1216a027ab33876e01ce95c52ae2d84b5f17a) for pshazz, which was also effected, by changing to import all libs from the `pshazz.ps1` entry script.

I didn't make patches for scoop because of the main bucket extraction event. And now I point out the issue to see if anyone has interest to fix this.

### LINKED ISSUE #4243: Using Scoop for private tools
I've searched around but have been unable to find a clear answer to this question. Can Scoop be configured to download artifacts from private (authenticated) URLs, such as a Github Release from a private repository)? Thanks!

### LINKED ISSUE #4784: [Bug] Shim path with umlauts is not encoded in utf-8
#### Current Behavior

Often when I try to run commands I receive an error because of the path. My username contains a space and umlauts. Unfortunately scoop does not seem to use utf-8 or so it seems. So instead of "á" in the username in the path I see a question mark instead which obviously results in an error. This is a current issue, I guess a month old.

#### Expected Behavior

I expect the path to be encoded correctly with umlauts as well.

#### Additional context/output

```
PS C:\Users\Peter Novám> scoop bucket list
Could not create process with command 'C:\Users\Peter Nov?k\scoop\apps\git\current\cmd\git.exe  -C "C:\Users\Peter Novßk\scoop\buckets\extras" log --format=%aD -n 1'.
Could not create process with command 'C:\Users\Peter Nov?k\scoop\apps\git\current\cmd\git.exe  -C "C:\Users\Peter Novßk\scoop\buckets\extras" config remote.origin.url'.
Could not create process with command 'C:\Users\Peter Nov?k\scoop\apps\git\current\cmd\git.exe  -C "C:\Users\Peter Novßk\scoop\buckets\games" log --format=%aD -n 1'.
Could not create process with command 'C:\Users\Peter Nov?k\scoop\apps\git\current\cmd\git.exe  -C "C:\Users\Peter Novßk\scoop\buckets\games" config remote.origin.url'.
Could not create process with command 'C:\Users\Peter Nov?k\scoop\apps\git\current\cmd\git.exe  -C "C:\Users\Peter Novßk\scoop\buckets\java" log --format=%aD -n 1'.
Could not create process with command 'C:\Users\Peter Nov?k\scoop\apps\git\current\cmd\git.exe  -C "C:\Users\Peter Novßk\scoop\buckets\java" config remote.origin.url'.
Could not create process with command 'C:\Users\Peter Nov?k\scoop\apps\git\current\cmd\git.exe  -C "C:\Users\Peter Novßk\scoop\buckets\nirsoft" log --format=%aD -n 1'.
Could not create process with command 'C:\Users\Peter Nov?k\scoop\apps\git\current\cmd\git.exe  -C "C:\Users\Peter Novßk\scoop\buckets\nirsoft" config remote.origin.url'.
Could not create process with command 'C:\Users\Peter Nov?k\scoop\apps\git\current\cmd\git.exe  -C "C:\Users\Peter Novßk\scoop\buckets\versions" log --format=%aD -n 1'.
Could not create process with command 'C:\Users\Peter Nov?k\scoop\apps\git\current\cmd\git.exe  -C "C:\Users\Peter Novßk\scoop\buckets\versions" config remote.origin.url'.

Name     Source Updated Manifests
----     ------ ------- ---------
extras                       1430
games                         215
java                          220
nirsoft                       266
versions                      278
```

### System details

**Windows version:** [e.g. 7, 8, 10]
Windows 11

**OS architecture:** [e.g. 32bit, 64bit]
64bit

**PowerShell version:** [output of `"$($PSVersionTable.PSVersion)"`]
```
Major  Minor  Build  Revision
-----  -----  -----  --------
5      1      22000  282
```

**Additional software:** [(optional) e.g. ConEmu, Git]
I also use conemu but in this case it happens also with pure PowerShell

#### Scoop Configuration

```json
{
    "lastupdate":  "2022-03-08T09:11:13.4293955+01:00",
    "SCOOP_REPO":  "https://github.com/lukesampson/scoop",
    "SCOOP_BRANCH":  "master"
}

```

### LINKED ISSUE #4790: [Bug] `scoop prefix` not working for global installs
## Bug Report
Starting [0.1.0](https://github.com/ScoopInstaller/Scoop/releases/tag/v0.1.0), `scoop prefix <app>` fails to find the path if the app is installed globally.

#### Current Behavior
`scoop prefix <app>` does not work if the app is installed globally.

#### Expected Behavior
It should work.

#### Additional context/output

```console
$ scoop list
Installed apps:

Name                       Version             Source                                                       Updated             Info
----                       -------             ------                                                       -------             ----
7zip                       21.07               main                                                         2022-03-04 14:03:47
scoop-completion           0.2.3               extras                                                       2022-03-04 14:03:53
aria2                      1.36.0-1            main                                                         2022-03-04 14:03:49 Global install
neovim                     0.6.1               main                                                         2022-03-04 14:03:52 Global install

$ scoop prefix neovim
Could not find app path for 'neovim'.
```

### System details

**Windows version:** [e.g. 7, 8, 10]
11

**OS architecture:** [e.g. 32bit, 64bit]
64bit

**PowerShell version:** [output of `"$($PSVersionTable.PSVersion)"`]
7.2.1

#### Scoop Configuration

```json
{
  "lastupdate": "2022-03-09T15:18:49.3892465+08:00",
  "SCOOP_REPO": "https://github.com/ScoopInstaller/Scoop",
  "SCOOP_BRANCH": "master",
  "aria2-warning-enabled": false
}
```

### LINKED ISSUE #4792: [Bug] scoop in docker shims seem broken
#### Current Behavior
install scoop in a docker container
install a scoop package (such as git)
use scoop package in a subsequent RUN command results in:
```
Could not create process with command '"C:\Users\ContainerAdministrator\scoop\apps\git\current\cmd\git.exe"  clone -q https://github.com/samuelcolvin/pydantic.git C:\Users\ContainerAdministrator\AppData\Local\Temp\pip-install-xmwtmaol\pydantic_4641e039e3774955b81c25db0d639f23'.
```

#### Expected Behavior
Can use installed programs

#### Possible Solution
https://github.com/ScoopInstaller/Scoop/tree/2021-12-26 does work

### System details

**Windows version:** 10

**OS architecture:** 64bit

**PowerShell version:** [output of `"$($PSVersionTable.PSVersion)"`]

**Additional software:** [(optional) e.g. ConEmu, Git]

#### Scoop Configuration

```json
//# Your configuration here
```

I will try to add more config details asap, but maybe someone already has an idea?

### LINKED ISSUE #4799: [Bug] [Dev] Skipping chores in logs not working
image

Feature added to `develop` branch in https://github.com/ScoopInstaller/Scoop/pull/4672

### LINKED ISSUE #4808: [Bug]
## Bug Report
When running scoop from git bash, using `'*'` (for example: `scoop update '*'`) expands to all files in the current directory, even though it's quoted correctly. I'm certain this used to work correctly the last time I tried it a few months ago.

#### Current Behavior
Running in git bash:
```
$ scoop update '*'
ERROR '3D' isn't installed.
ERROR 'AppData' isn't installed.
ERROR 'Application' isn't installed.
ERROR 'Contacts' isn't installed.
ERROR 'Cookies' isn't installed.
ERROR 'Desktop' isn't installed.
ERROR 'Documents' isn't installed.
ERROR 'Downloads' isn't installed.
...
```

#### Expected Behavior
Running `scoop update '*'` from git bash updates all out of date applications.

#### Additional context/output

I tracked it down to `~/scoop/shims/scoop` shell script, which currently uses `$@` instead of `"$@"` to pass all arguments to powershell. This sh->ps1 shim seems to be written from here:

https://github.com/ScoopInstaller/Scoop/blob/7d5a47cc774961c15fdae6be1daf6331282b0161/lib/core.ps1#L689-L695

The other sh shims in lib/core.ps1 (sh->cmd/bat/jar/py/...) all use `"$@"`, so I believe it was omitted unintentionally.

#### Possible Solution
Fix the quoting of `$@` in the above lines in lib/core.ps1

### System details

**Windows version:** 10

**OS architecture:** 64bit

**PowerShell version:**
```
PS> $PSVersionTable.PSVersion
Major  Minor  Build  Revision
-----  -----  -----  --------
5      1      19041  1320
```

**Additional software:**
```
$ git --version
git version 2.35.1.windows.2
$ bash --version
GNU bash, version 4.4.23(2)-release (x86_64-pc-msys)
```

#### Scoop Configuration
```json
{
    "lastupdate":  "2022-03-13T07:24:36.8369155+03:00",
    "SCOOP_REPO":  "https://github.com/lukesampson/scoop",
    "SCOOP_BRANCH":  "master",
    "alias":  {

              }
}
```

### LINKED ISSUE #4810: [Bug] `scoop download` succeeds even when no free space
## Bug Report
Scoop reports that download was successful when it fails due to lack of free space.

#### Current Behavior

When nearly out of free space on disk and running `scoop download` to get a file, once all free space is used, Scoop stops downloading and says both that it failed to download and that it succeeded.

#### Expected Behavior

If the download fails midway it should fail. No green text should appear.

#### Additional context/output

```
❯ scoop download testing/coq -a 32bit
INFO  Starting download for coq...
Coq-Platform-8.14.2022.01-installer-windows-i686_signed.exe (318.5 MB) [=========>  ]  84%
There is not enough space on the disk. : 'C:\Users\me\scoop\cache\coq#2022.01.0#https_github.com_coq_platform_releases_download_2022.01.0_Coq-Platform-8.14.2022.01-installer-windows-i686_signed.exe_dl.7z.download'
ERROR URL https://github.com/coq/platform/releases/download/2022.01.0/Coq-Platform-8.14.2022.01-installer-windows-i686_signed.exe#/dl.7z is not valid
'coq' (2022.01.0) was downloaded successfully!         <-- in green
```

#### Possible Solution

I'd love to try making a PR but have no idea where to start or how to test!

### System details

**Windows version:** 11

**OS architecture:** 64bit

**PowerShell version:** 7.2.1

**Additional software:** none relevant

#### Scoop Configuration

```
   1   │ {
   2   │   "SCOOP_BRANCH": "master",
   3   │   "lastupdate": "2022-03-14T23:16:07.9419731+00:00",
   4   │   "SCOOP_REPO": "https://github.com/lukesampson/scoop",
   5   │   "debug": false,
   6   │   "aria2-enabled": false,
   7   │   "aria2-retry-wait": 4,
   8   │   "aria2-split": 16,
   9   │   "aria2-max-connection-per-server": 16,
  10   │   "aria2-min-split-size": "4M",
  11   │   "alias": {
  12   │     "reinstall": "scoop-reinstall"
  13   │   }
  14   │ }
```

### LINKED ISSUE #4819: [Bug] scoop reset failed for global installed apps
#### Current Behavior
Here are some JDKs installed globally by scoop:
图片

when i use `scoop reset temurin11-jdk`, it saids `ERROR 'temurin11-jdk' isn't installed`

#### Expected Behavior
Successfully reset shims and environment variables

### System details

**Windows version:** 10

**OS architecture:** 64bit

**PowerShell version:** 5.1.19041.1320

#### Scoop Configuration

```json
{
    "lastupdate":  "2022-03-17T08:04:45.0044426+08:00",
    "SCOOP_REPO":  "http://github.com/ScoopInstaller/Scoop",
    "SCOOP_BRANCH":  "master",
    "aria2-enabled":  false,
    "aria2-config":  false,
    "proxy":  "127.0.0.1:10809"
}
```

### LINKED ISSUE #4829: Gitter badge in README points to 404
The Gitter room hasn't been migrated yet and still points to https://gitter.im/lukesampson/scoop, but the README has been updated to point to https://gitter.im/ScoopInstaller/Scoop, which doesn't exist yet.

### LINKED ISSUE #4833: [Bug] bug in scoop reset
In libexec/scoop_reset.ps1 , line 42
I think you need to pass the 2nd argument to the function ```installed``` in the if condition as ``` (installed $app $global)``` in order to correctly reset globally installed package. otherwise it just exits with error saying "ERROR pkg isn't installed"
```
    $app, $bucket, $version = parse_app $app

    if(($global -eq $null) -and (installed $app $true)) {
        # set global flag when running reset command on specific app
        $global = $true
    }

    if($app -eq 'scoop') {
        # skip scoop
        return
    }

    if(!(installed $app)) { // BUG HERE
        error "'$app' isn't installed"
        return
    }

    if ($null -eq $version) {
        $version = Select-CurrentVersion -AppName $app -Global:$global
    }
   ```

### System details
**Windows version:** [e.g. 7, 8, 10]

**OS architecture:** [e.g. 32bit, 64bit]

**PowerShell version:** [output of `"$($PSVersionTable.PSVersion)"`]

**Additional software:** [(optional) e.g. ConEmu, Git]

#### Scoop Configuration

```json
//# Your configuration here
```

### LINKED ISSUE #4835: [Bug] misspelling?
#### Current Behavior
```
> scoop help shim
Usage: scoop shim <subcommand> [<shim_names>] [<command_path> [<args>...]] [-g(lobal)]

Manipulate Scoop shims: add, rm, list, info, alter, etc.

To add a costom shim, use the 'add' subcommand:

    scoop shim add <shim_name> <command_path> [<args>...]

To remove a shim, use the 'rm' subcommand (CAUTION: this could remove shims added by an app manifest):

    scoop shim rm <shim_names>

To list all shims or matching shims, use the 'list' subcommand:

    scoop shim list [<shim_names>]

To show a shim's information, use the 'info' subcommand:

    scoop shim info <shim_name>

To alternate a shim's target source, use the 'alter' subcommand:

    scoop shim alter <shim_name>

Options:
  -g(lobal)       Add/Remove/Info/Alter global shim(s)
                  (NOTICE: USING SINGLE DASH)
                  (HINT: To pass arguments like '-g' or '-global' to the shim, use quotes)
```
notice how it says "costom"

#### Expected Behavior
it's spelled "custom"

### System details

**Windows version:** 10

**OS architecture:** 64bit

**PowerShell version:** 7.2.2

**Additional software:** 

#### Scoop Configuration

irrelevant

### LINKED ISSUE #4841: [Bug] scoop prefix can not find app path for any package
image

### LINKED ISSUE #4863: [Feature] The excavator GitHub action run does not report error if something fails to update
#### Current Behavior

Take a look at the "Excavate" step of this GitHub Action run: https://github.com/ScoopInstaller/Extras/runs/5963485483. There are plenty of errors that some apps fail to update. However, the whole run completed successfully. If no one notices these hidden errors, some apps may never get auto updated. This is not big problem of large repo like Scoop/Extras, but could be for people that maintain personal bucket.

#### Expected Behavior

If the current behavior is expected, that for some reason you guys don't want error from the action run, please provide an option so that personal bucket maintainers can get notification when action fails.

#### Additional context/output

Not sure if this issue belongs here or https://github.com/ScoopInstaller/GithubActions.

#### Possible Solution

Is this because https://github.com/ScoopInstaller/GithubActions/blob/main/src/ActionWrapper.psm1 `Invoke-Action` does not propagate the error from [autoupdate.ps1](https://github.com/ScoopInstaller/Scoop/blob/master/lib/autoupdate.ps1#L578)?

### System details

**Windows version:** 11

**OS architecture:** 64bit

**PowerShell version:** 7.2.2

#### Scoop Configuration

```json
{
  "SCOOP_REPO": "https://github.com/ScoopInstaller/Scoop",
  "SCOOP_BRANCH": "master",
  "shim": "kiennq",
  "ignore_running_processes": true,
  "aria2-warning-enabled": false
}
```

### LINKED ISSUE #4868: [Feature] Do not use the Unlicense for scoop
## Feature Request

**Note: This is not legal advice. I am not a lawyer.**
#### Is your feature request related to a problem? Please describe.

Scoop is currently using the Unlicense and was using it since development of it began (that means Scoop is released to the public domain). I have an issue with Scoop's license - the Unlicense is **invalid** in some countries (e.g. Germany), and does not seem to be clear in a few countries (e.g. Australia). This is a problem with the Unlicense, so we should be moving to MIT or Apache because "public domain" does not exist everywhere. Also, [the Unlicense is not allowed to be used inside Google](https://opensource.google/docs/patching/#forbidden), so Google would not be allowed to make contributions to Scoop or fork it to @google. However, Google employees are important contributors to open-source, so always stay away from the Unlicense. There are also rumours going on in github/choosealicense.com#805.

This should apply to all Scoop repositories using the Unlicense (yes, that means most of the repositories).

Note that [0BSD](https://choosealicense.com/licenses/0bsd) is an exception.
#### Describe the solution you'd like

I would edit the `LICENSE` file to use a permissive license instead of the Unlicense or any other public domain dedication.
#### Describe alternatives you've considered

Using Chocolatey instead. I am also writing [CPPM](https://github.com/Tyler887/CPPM), a package manager not only for Windows, but for _any operating system that supports Python_. It's not released into the public domain but is under a permissive license.

### LINKED ISSUE #4887: [Feature] Add --all switch to "scoop cleanup" subcommand.
## Feature Request

Make "scoop cleanup --all" available, in the same way as "scoop update --all" is currently available.

#### Is your feature request related to a problem? Please describe.

I am trying to use msys2 bash as a main shell on Windows, and encountered this 
error:
```
$ scoop update '\*'
Compare-Version : Cannot process argument transformation on parameter 'ReferenceVersion'. Cannot convert value to type System.String.
At C:\Users\marti\scoop\apps\scoop\current\lib\core.ps1:353 char:68
+ ... utdated = ((Compare-Version -ReferenceVersion $status.version -Differ ...
+                                                   ~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Compare-Version], ParameterBindingArgumentTransformationException
    + FullyQualifiedErrorId : ParameterArgumentTransformationError,Compare-Version

\*: 21.07 0.2.26 1.36.0-1 0.20.0 0.6.8 2022-03-29 3.23.1 1.47.0 7.82.0_4 3.6 0.8.0 5.0.1 5.38 11.2.0 2.36.0.windows.1 1.18.1 3.4.4 1.45.2 2.3.6 3.0.0 6.2.5 0.9.0 1.9 0.50 1.6 1.1.2 0.21.0 4.3 3.20.4 2022-03-19 6.2-16 1.10.2 7.92 18.0.0 0.61.0 3.0.2 3.2.0.0 8.1.5 1.8.11 5.0.14.1 0.5.0 13.0.0 3.38.2 1.6.2 0.2020.01.26 4.11 1.19.2 0.22.0 2.2.1 0.8.1 20220319-142410-0fcdea07 1.21.3 2021.12.17 (latest version)
Latest versions for all apps are installed! For more information try 'scoop status'
```
But then I found that "scoop update --all" works correctly. But after that, I wanted to clean the scoop cache but same problem there. But the "scoop cleanup" subcommand lacks a --all alias.

#### Describe the solution you'd like
Add --all, -a flags to the "scoop cleanup" subcommand.

```
  -a, --all                 Update all apps (alternative to '*')
```

### PR DESCRIPTION
**Pull request by niheaven**
### BREAKING CHANGES

- The license of Scoop has changed to dual-license Unlicense or MIT (#4903)
- Rename `checkver_token` to `gh_token` and `SCOOP_CHECKVER_TOKEN` to `SCOOP_GH_TOKEN` (#4832, #4842)

### Features

- **relicense:** Relicense to dual-license (Unlicense or MIT) (#4903, #4870)
- **install:** Allow downloading from private repositories (#4254)
- **scoop-cleanup:** Add -a/--all switch to cleanup all apps (#4906)

### Bug Fixes

- **bucket:** Return empty list correctly in `Get-LocalBucket` (#4885)
- **shim:** Manipulating shims with UTF8 encoding (#4791, #4813)
- **shim:** Correctly quote $@ in sh->ps1 shims (#4809)
- **install:** Fix issue with installation inside containers (#4837)
- **installed:** If no `$global`, check both local and global installed (#4798)
- **update:** Skip logs starting with `(chore)` (#4800)
- **scoop-download:** Add failure check (#4822)
- **scoop-list:** Fix date in 'Updated' column showing the months in the place of minutes (#4880)
- **scoop-prefix:** Fix typo that breaks global installed apps (#4795)

### Performance Improvements

- **scoop:** Load libs only once (#4839, #4884)

### Code Refactoring

- **bucket:** Move 'Find-Manifest' and 'list_buckets' to 'buckets' (#4814)
- **relpath:** Use `$PSScriptRoot` instead of `relpath` (#4793)
- **reset_aliases:** Move core function of `reset_aliases` to `scoop` (#4794)
- **config:** Rename checkver_token to gh_token and SCOOP_CHECKVER_TOKEN to SCOOP_GH_TOKEN (#4832, #4842)

### Builds

- **checkver:** Add option to throw error as exception (#4867)
- **schema:** Remove 'description' from required fields (#4853, #4874)

### Documentation

- **changelog:** Rearrange CHANGELOG (#4897)
- **readme:** Update installation instruction (#4825)
- **readme:** Fix badges for Gitter and CI Tests (#4830)
- **scoop-shim:** Fix typo (#4836)

## Close Issues

- Closes #4243
- Closes #4784
- Closes #2573
- Closes #2230
- Closes #4808
- Closes #4792
- Closes #4819
- Closes #4833
- Closes #4841
- Closes #4799
- Closes #4810
- Closes #4790
- Closes #3418
- Closes #2123
- Closes #4863
- Closes #4829
- Closes #4835
- Closes #4887
- Closes #4868

## Credits

Thanks to the following contributors (ordered by PR merging time):

@niheaven @rashil2000 @chawyehsu @e6c31d @tech189 @tmsmith @astelmachonak-nydig @kingster @kiennq @CrendKing @cskrisz @lewis-yeung

### CODE REVIEW

**Comment by astelmachonak-nydig** [comment]
I created PR to GithubActions repo, needed for the new version due to the breaking change with config parameter rename: https://github.com/ScoopInstaller/GithubActions/pull/7

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,44 @@
...
+### Builds
+
+- **checkver:** Add option to throw error as exception ([#4863](https://github.com/ScoopInstaller/Scoop/issues/4863))
+- **schema:** Remove 'description' from required fields ([#4853](https://github.com/ScoopInstaller/Scoop/issues/4853), [#4874](https://github.com/ScoopInstaller/Scoop/issues/4874))
+
+### Documentation
+
+- **readme:** Update installation instruction ([#4825](https://github.com/ScoopInstaller/Scoop/issues/4825))
+- **readme:** Fix badges for Gitter and CI Tests ([#4830](https://github.com/ScoopInstaller/Scoop/issues/4830))
+- **scoop-shim:**: Fix typo ([#4836](https://github.com/ScoopInstaller/Scoop/issues/4836))
```
Comment (tech189) [inline CHANGELOG.md:40]:
```suggestion
- **scoop-shim:** Fix typo ([#4836](https://github.com/ScoopInstaller/Scoop/issues/4836))
```
There's an extra semicolon here 😅

Comment (niheaven) [reply CHANGELOG.md:40]: Aha, yes, should be fixed in other PR (maybe relicense one?)

**Comment by rasa** [comment]
Can we consider merging either #4870 or #4892 to dual license with MIT, per the discussion in #4868?

**Comment by niheaven** [comment]
> Can we consider merging either #4870 or #4892 to dual license with MIT, per the discussion in #4868?

That's what I'm thinking of, and should we vote for which one is to be merged?

**Comment by rasa** [comment]
> That's what I'm thinking of, and should we vote for which one is to be merged?

Sure. #4870 is mergable once the strikeout text is cleaned up. I tried fixing #4870 but GitHub's UI wouldn't let me, which spurred me to create #4892, not to negate @Tyler887's work, but simply to express a preference.

Here's the poll:

[](https://api.gh-polls.com/poll/01G2A86GXYEFVYQRSY8R10A9M7/4870%20-%20chore%3A%20Dual%20license%20between%20Unlicense%20and%20MIT%20-%20%40Tyler887/vote)
[](https://api.gh-polls.com/poll/01G2A86GXYEFVYQRSY8R10A9M7/4892%20-%20chore%3A%20Dual%20license%20via%20MIT%20license%20and%20Unlicense%20%20-%20%40rasa/vote)

**Comment by chawyehsu** [comment]
I saw there are insignificant commits adding to #4870 which make no sense to me at all. If @TylerHaigh has no intention of fixing them, including the strikeout problem, I would vote #4892 but it needs tweaks. Basically I prefer one merged license file over two seperated licenses at this point.

**Comment by niheaven** [comment]
Reimplement #4870 via #4903, if one file is better, we could edit there, merge it and move on.

**Comment by rasa** [comment]
#4903 works for me, thx!

**Review by rashil2000 — APPROVED** [review]

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**